coord/src/info_command.c

```diff
@@ -32,7 +32,8 @@ static InfoFieldSpec toplevelSpecs_g[] = {
     {.name = "offset_bits_per_record_avg", .type = InfoField_DoubleAverage},
     {.name = "indexing", .type = InfoField_WholeSum},
     {.name = "percent_indexed", .type = InfoField_DoubleAverage},
-    {.name = "hash_indexing_failures", .type = InfoField_WholeSum}};
+    {.name = "hash_indexing_failures", .type = InfoField_WholeSum},
+    {.name = "number_of_uses", .type = InfoField_Max}};
```

Comment:
@K-Jo WDYT?

Reply:
We can also consider to count reads and writes separately, e.g., `FT.SEARCH` is a read operation and `ALIASADD` is a write operation (`number_of_reads`, `number_of_writes`)

I think we should not count DEBUG command usages, e.g., TTL

Reply:
What about adding last access time (timestamps), so for example, in `INFO` command we could ask for the 5 most used indices, and the 5 most recently used indices.

In addition, I suggest to commit massive formatting changes in a separate commit than actual code changes.